cmd/ingress/ingress.go

```diff
@@ -96,6 +97,8 @@ the apisix cluster and others are created`,
 			log.DefaultLogger = logger
 			log.Info("apisix ingress controller started")
 
+			log.Info("version: ", version.Short())
```

Comment:
I think here output the long version is better :)

Reply:
OK，I will deal with this